modules/gltf/gltf_document.cpp

```diff
@@ -6595,12 +6611,6 @@ Error GLTFDocument::_parse(Ref<GLTFState> state, String p_path, FileAccess *f, i
 	state->major_version = version.get_slice(".", 0).to_int();
 	state->minor_version = version.get_slice(".", 1).to_int();
 
-	for (int32_t ext_i = 0; ext_i < document_extensions.size(); ext_i++) {
-		Ref<GLTFDocumentExtension> ext = document_extensions[ext_i];
-		ERR_CONTINUE(ext.is_null());
-		err = ext->import_preflight(this);
-		ERR_FAIL_COND_V(err != OK, FAILED);
-	}
 	err = _parse_gltf_state(state, p_path, p_bake_fps);
```

Comment:
It would be nice to maintain a hook here, before `_parse_gltf_state`, perhaps in addition to the other `import_preflight(GLTFState)` function.

We could call it `_import_pre_parse(State)` or `_import_pre_parse(Dictionary)` where you pass in `state->json` and allow mutation... or something like that

Reply:
Revised code:

* use `_import_preflight` which is after the parse into json.
* use `_import_pre_parse` which is after the parse into gltf

Reply:
Looks great.